.github/workflows/nextjs.yml

```diff
@@ -70,10 +76,6 @@ jobs:
         run: ${{ steps.detect-package-manager.outputs.manager }} ${{ steps.detect-package-manager.outputs.command }}
       - name: Build with Next.js
         run: ${{ steps.detect-package-manager.outputs.runner }} next build
-      - name: Add .nojekyll file
```

Comment:
I guess we can merge with this change about the .nojekyll file, but I'm thinking that might be needed?